cluster-autoscaler/cloudprovider/gce/gce_price_model.go

```diff
@@ -283,9 +307,27 @@ func (model *GcePriceModel) NodePrice(node *apiv1.Node, startTime time.Time, end
 			price = price * preemptibleDiscount
 		}
 	}
-	// TODO: handle SSDs.
 
-	price += getAdditionalPrice(node.Status.Capacity, startTime, endTime)
+	// GPUs
+	if gpuRequest, found := node.Status.Capacity[gpu.ResourceNvidiaGPU]; found {
+		gpuPrice := gpuPricePerHour
+		if node.Labels != nil {
```

Comment:
nit: can we log sth about fallback to default pricing if there's no label? (or error if this isn't supposed to happen)

Reply:
I'm not sure about this - in that case falling back to the default seems to be the clearly right thing to do (as opposed to situation where we don't have pricing info for some hardware). Those logs will get quite spammy (as it's likely to be repeated many times each loop) and not having any labels on node is unexpected, but not necessarily wrong.